### ISSUE TITLE
7Span: Story Detail Page: Change requests 8/28/23 but 1 Bug 9/27/23

### ISSUE THREAD

**Issue by WendyLBelcher** [PrincetonPEMM/7Span-Phase2#126]
### Technical Information Box (left sidebar)
- [x] Change comma to semicolon in `Earliest Manuscripts in which Story Appears:` 
So, B-Mc (McCarthy) 1574; ...
![align sidebar](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/b72c03f4-3d39-41db-9a7e-a4790eed2f10)

**Comment by WendyLBelcher** [comment]
- [ ] Change the wording of TÃ¤Ë€ammÉ™rÃ¤ to Ta'ammera
![screenshot underscore](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/f623cd13-2280-46cd-86e2-4927da0aacac)

**Comment by WendyLBelcher** [comment]
- [ ] Show manuscripts with See More
The maximum # of manuscripts that should show in total for each century is 7. If there are more than 7 manuscripts per century, please include a "See more" link that will display all manuscripts from that century. We are open to suggestions around the 7 number so that it won't slow down the page. 

![screenshot manuscripts](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/9819f3bf-7825-442a-ace5-b51a9582fad8)

**Comment by WendyLBelcher** [comment]
### Info Box
- [ ] Display `Canonical Story` table fields `canonical_incipit`  and (if not empty) `canonical_incipit_2` and (if not empty) `canonical_incipit_3`
- [ ] Display with each incipit the following: `manuscript `and `scan_start` (unless it is empty: then use `folio_start`)

Display in this format:
<incipit> From <manuscript>, s. <scan_start> or
<incipit> From <manuscript>, f. <folio_start>

Example, for PEMM ID 636:
Incipit(s): ወኮነ፡ እምድኅረ፡ ዕርገቱ፡ ለእግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ ሎቱ፡ ስብሐት፡ በክልኤቱ፡ ዓመት፡ ወዘይበዝኅ፡ እምኔሁ፡ ወድኅረ፡ ብስራተ፡ ወንጌል፡ ቅዱስ፡ ተንሥኡ፡ ዓላውያን፡ ነገሥት፡ ወዓለዉ፡ ሃይማኖተ፡ ወነፈቅዋ፡ ለቤተ፡ ክርስቲያን፡ ወሰደዱ፡ መሃይምናነ፡ እለ፡ ኢአመኑ፡ በዕልወቶሙ From VL-Vatican (BAV) 298, f. 120v

**Comment by WendyLBelcher** [comment]
Ignore for now:
### Content Information:
If pemm_theme_id_number has >.00 (e.g., .01., 02.)

- [ ] display this information in this wording (according the numbers after the decimal point)
Unit: Part [] of [] parts of this story 
Unit: Part 2 of 7 parts of this story (if .02 and highest is .07)

**Comment by WendyLBelcher** [comment]
- [ ] Fix diacritics
![diacritics BM](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/a4e43ac1-c69d-4822-93e5-544bddfe2719)

**Comment by WendyLBelcher** [comment]
Ignore for now:
### Content Information:
If pemm_theme_id_number has >.00 (e.g., .01., 02.)

- [ ] display this information in this wording (according the numbers after the decimal point)
Unit: Part [] of [] parts of this story 
For example:
Unit: Part 2 of 7 parts of this story (if .02 and highest is .07)

**Comment by WendyLBelcher** [comment]
- [ ] Fix Cite This Translation when `Translation `table `translation_status` =Published Translation
This is wrong:
![cite this](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/9e87f73c-1cd1-4c81-b2e0-a4daa132c82c)

**Comment by WendyLBelcher** [comment]
OTHER TRANSLATIONS & EDITIONS OF THIS STORY (middle of page)
- [ ] If possible, spell out TGS as the title if it appears in the `Translation `table `manuscript` field for OTHER TRANSLATIONS & EDITIONS OF THIS STORY 
For
Our TG (EOTC) 1931/1968. My English translation is [The Miracles of Mary in Ethiopic and Amharic: 111 Miracles].
Our TGS (EOTC) 1994. My English translation is [Sixty-four Miracles of Mary in Ethiopic and Amharic]
Our TGS (EOTC) 1996. My English translation is [The Miracles of Mary in Ethiopic and Amharic: Part Two: 402 /381/ Miracles]. 
Our TGS (EOTC) 2014. My English translation is [The Miracles of Mary in Ethiopic and Amharic: 346 Miracles]. 
Our TGS (EOTC) 1971. My English translation is [The Miracles of Mary in Ethiopic and Amharic: Part One: 270 Miracles].

**Comment by WendyLBelcher** [comment]
Ignore for ow
See painting by story page

**Comment by WendyLBelcher** [comment]
### Paintings Detail page link
- [ ] Add link to painting detail page on each painting in the carousel 
Note: 
Do NOT change what happens when a user clicks on a painting in the band of paintings below the carousel
Do NOT change what happens when a user clicks on a painting in the carousel
Do NOT change what happens when a user clicks on [] to expand the painting
BUT, if they click on the expanded painting, DO take them to that painting's paintings detail page
![link to painting detail carousel](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/2cd76f88-b2a3-4420-9e04-5469d466f547)

**Comment by WendyLBelcher** [comment]
### Ignore
- [ ] Spell out an acronym here (if possible)
![colon lang](https://github.com/PrincetonPEMM/7Span-Phase2/assets/56000106/a8d8f379-e369-4ebd-a8ae-78c921a67c06)

 If **TGS (EOTC)...** appears in the `Translation `table `manuscript` field for OTHER TRANSLATIONS & EDITIONS OF THIS STORY 
do the following, if possible:
- [ ] If "From TG (EOTC) 1931/1968," put instead: From [The Miracles of Mary in Ethiopic and Amharic: 111 Miracles] (1931/1968),
- [ ] If "From TGS (EOTC) 1971,"        put instead: From [The Miracles of Mary in Ethiopic and Amharic: Part One: 270 Miracles] (1971), 
- [ ] If "From TGS (EOTC) 1994,"        put instead: From [The Miracles of Mary in Ethiopic and Amharic: 64 Miracles] (1994),
- [ ] If "From TGS (EOTC) 1996,"        put instead: From [The Miracles of Mary in Ethiopic and Amharic: Part Two: 402 Miracles] (1996), 
- [ ] If "From TGS (EOTC) 2014,"        put instead: From [The Miracles of Mary in Ethiopic and Amharic: 346 Miracles] (2014),

**Comment by WendyLBelcher** [comment]
Doing the above knocked out the good programming for the regular translations.

- [ ] Keep new programming
- [ ] But, correct back to original programming for `Translation` table `translation_status` does NOT = Published Translation

It wrongly has this, for example:
Jeremy Brown. 2022. "ID 125: The story of the cannibal from Qemer." , edited by Jeremy Brown. Updated by PEMM Copyeditor Taylor Eggan. From EMML (HMML) 9002, ff. 279r–281v. https://main.d2jda7ye5cy60r.amplifyapp.com/stories/125.
So, it wrongly lost our title, our editors, the date, "Last modified:", and it wrongly added "Updated by PEMM Copyeditor Taylor Eggan" All changes we requested ONLY apply to the two conditions. Under the other conditions, everything should stay the same. It was correct as it was before.
